WebserviceFileDataObject doesn't follow redirects

**Describe the bug**
WebserviceFileDataObject don't work if calling the URL gives a Return Code of 301 (redirect). 

**To Reproduce**
Using the following DataObject
` opendata-swiss-5g {
    type = WebserviceFileDataObject
    webserviceOptions {
      url = "http://data.geo.admin.ch/ch.bakom.mobil-antennenstandorte-5g/data/ch.bakom.mobil-antennenstandorte-5g_en.json"
    }
  }`
Results in

> Caused by: scala.MatchError: HttpResponse([B@162f9f37,301,Map(Connection -> Vector(keep-alive), Content-Length -> Vector(183), Content-Type -> Vector(text/html), Date -> Vector(Wed, 27 May 2020 14:41:27 GMT), Location -> Vector(https://data.geo.admin.ch/ch.bakom.mobil-antennenstandorte-5g/data/ch.bakom.mobil-antennenstandorte-5g_en.json), Server -> Vector(CloudFront), Status -> Vector(HTTP/1.1 301 Moved Permanently), Via -> Vector(1.1 59574f77a7cf2d23d64904db278e5711.cloudfront.net (CloudFront)), X-Amz-Cf-Id -> Vector(5bNnvgUVTLKMWJs3bZ5IVu6bpSCEnfpLh6eg8aAEuSTSCE-BNP65UQ==), X-Amz-Cf-Pop -> Vector(FRA53), X-Cache -> Vector(Redirect from cloudfront))) (of class scalaj.http.HttpResponse)
	at io.smartdatalake.util.webservice.DefaultWebserviceClient.checkResponse(DefaultWebserviceClient.scala:66)
	at io.smartdatalake.util.webservice.DefaultWebserviceClient.get(DefaultWebserviceClient.scala:49)
	at io.smartdatalake.workflow.dataobject.WebserviceFileDataObject.createInputStream(WebserviceFileDataObject.scala:124)


**Expected behavior**
The Webservice call receives a 301 because the file permanently redirected to the https:// equivalent of the URL:
https://data.geo.admin.ch/ch.bakom.mobil-antennenstandorte-5g/data/ch.bakom.mobil-antennenstandorte-5g_en.json
The file should be fetched from the new location.

--- a/src/main/scala/io/smartdatalake/util/webservice/DefaultWebserviceClient.scala
+++ b/src/main/scala/io/smartdatalake/util/webservice/DefaultWebserviceClient.scala
@@ -20,7 +20,7 @@ package io.smartdatalake.util.webservice
 
 import io.smartdatalake.config.ConfigurationException
 import io.smartdatalake.util.misc.SmartDataLakeLogger
-import scalaj.http.{Http, HttpRequest, HttpResponse}
+import scalaj.http.{Http, HttpOptions, HttpRequest, HttpResponse}
 
 import scala.util.{Failure, Success, Try}
 
@@ -145,6 +145,7 @@ private[smartdatalake] object DefaultWebserviceClient extends SmartDataLakeLogge
   def buildRequest(url: String): HttpRequest = {
     logger.debug(s"Building HttpRequest with url: <$url>")
     Http(url)
+      .option(HttpOptions.followRedirects(true))
   }
 
   /**

